texastribune/files
# Bump elements version for touch listeners

#### What's this PR do?

- Bumps `elements` version. See https://github.com/texastribune/Elements/pull/7

#### Related PRs

texastribune: https://github.com/texastribune/texastribune/pull/4677
tt-docker-base: https://github.com/texastribune/tt-docker-base/pull/599
elements: https://github.com/texastribune/Elements/pull/7

#### Why are we doing this? How does it help us?

See https://github.com/texastribune/Elements/pull/7

#### How should this be manually tested?

_In the docs_

1. Go to <https://texastribune.github.io/Elements/> on your mobile device
2. Click on "User Interface"
3. Click on the gear to open the context menu
4. Click on "Add File" and choose a file to upload
5. Try clicking on the add button. Note that the add button is unresponsive
6. Clone this repository and install dependencies with `npm install`
7. Generate new docs by running `npm run doc`
8. Spin up a local server at the `docs` directory
9. Run `ifconfig` to get your local network IP address
10. Go to the user interface tutorial
11. Note that uploading a file works now
12. Note that the expected behavior continues to work on desktop

#### How should this change be communicated to end users?

Send a note in #team-engineering

#### Are there any smells or added technical debt to note?

Not related to this PR, but the buttons are almost impossible to touch because they're so small on mobile. And, zooming in to get a bigger target will sometimes make the dialog show up offscreen.

#### What are the relevant tickets?

<https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwPxWENGdckjxRrP/recvM0Tfl61zz19kp?blocks=hide>

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***

* [x] Tested manually on mobile? *(See manual testing)*

#### TODOs / next steps:

* [x] After the elements PR is merged, change the elements branch name to a version number
* [x] Bump the version number
* [x] Run `npm run doc`
